.travis.yml

```diff
@@ -20,3 +20,18 @@ matrix:
         - pip install coveralls
       env: TOXENV=cover
       after_success: coveralls
+    - language: node_js
+      node_js: "4.2.6"
+      services:
+        - docker
+      install:
+        - docker build -t ceagle .
+        - docker run -d -p 127.0.0.1:5000:5000 ceagle
+        - git clone http://github.com/cybertk/abao
```

Comment:
I tried this however it seems that npm on travisci ignores dependencies installation